.travis.yml

```
@@ -1,36 +1,58 @@
-branches:
-  only:
-    - master
-    - /^v?[0-9]/
+language: cpp
 
-language: objective-c
+os:
+- linux
+- osx
+- windows
+
+dist: trusty
+
+addons:
+  apt:
+    packages:
+    - libudev-dev
 
 env:
+  global:
+  - PREBUILD_TARGETS="6.0.0 8.0.0 10.0.0 11.0.0"
```

```suggestion
  - PREBUILD_TARGETS="6.0.0 8.0.0 10.0.0 11.0.0"
  - PREBUILD_ELECTRON_TARGETS="3.0.0 4.0.0 4.0.4"
```

This is the first half of adding Electron support to the prebuild targets.